package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
To make pre-commit work I needed to move husky to the root folder within the monorepo. The pre-commit did not trigger when the setup was done within _/frontend_ folder.